src/interface/sparse_pack_base.cpp

```diff
@@ -232,6 +239,10 @@ SparsePackBase SparsePackBase::Build(T *pmd, const PackDescriptor &desc,
                       pack_h(3, b, idx) = pv->flux[X3DIR].Get(0, t, u, v);
                     }
                   }
+                  for (auto el :
+                       GetTopologicalElements(pack_h(0, b, idx).topological_type)) {
+                    pack_h(static_cast<int>(el) % 3, b, idx).topological_element = el;
+                  }
```

Comment:
I probably not familiar enough with the new topological element approach, but is the `%3` guaranteed correct for all cases that loop is called for?

Reply:
Yeah, it is correct for all cases because of the way the `TopologicalElement` enum  is defined.